README.md

```diff
@@ -169,6 +169,27 @@ If you get a `Error:Cannot run program "bazel" (in directory "$path"): error=2,
 - Click "Apply" then "OK"
 - Re-import the project by hitting the bazel logo in the upper right hand corner.
 
+### IntelliJ
+
+- Install IntelliJ. Community Edition is sufficient.
+- Open Android Studio and install the Bazel extension.
```

Comment:
So you need both IntelliJ and Android Studio?

Reply:
Oops :-|